contracts/src/L1/rollup/ScrollChain.sol

```diff
@@ -292,15 +292,15 @@ contract ScrollChain is OwnableUpgradeable, PausableUpgradeable, IScrollChain {
         bytes32 _prevStateRoot,
         bytes32 _postStateRoot,
         bytes32 _withdrawRoot,
-        bytes calldata _aggrProof
+        bytes calldata
```

Comment:
we should add an additional method that can finalize without proof on Sepolia. We will still finalize with proofs for some but not all batches.

Reply:
Then we also need to change rollup relayer, right?